src/mam4xx/conversions.hpp

```diff
@@ -19,6 +19,8 @@ using Constants = haero::Constants;
 using haero::cube;
 using haero::square;
 
+constexpr Real mol_to_kmol = 1e-3;
+
 /// Given a number concentration for a species or mixture [m-3], computes and
```

Comment:
I could be wrong, but I think we have a few terms mixing in here, and we have to be certain of each one's units.

"stuff" can be number, volume, or mass
- mixing ratio : amount of stuff per kg dry air
- concentration: volume of stuff per m3 dry air

A volume mixing ratio is not a number concentration; vmr is m3 aerosol per kg dry air, so at minimum this comment needs to be changed.  Perhaps the function name, too; perhaps the function's content.

Reply:
I have no reason not to accept the definitions you give here, since I've only seen these terms used in the context of my work here and accepted the calculations in these functions as correct (oops). I will rewrite the functions in terms of those corrected quantities and tag you for a once-over when I commit.